Content.IntegrationTests/Tests/PostMapInitTest.cs

```diff
@@ -27,42 +27,135 @@ namespace Content.IntegrationTests.Tests
+            "/Maps/_NF/Shuttles/stellaris.yml",
+            "/Maps/_NF/Shuttles/stratos.yml",
+            "/Maps/_NF/Shuttles/vagabond.yml",
+            "/Maps/_NF/Shuttles/waveshot.yml",
+            // Admin
+            "/Maps/_NF/Shuttles/Admin/fishbowl.yml",
+            // Black Market
+            "/Maps/_NF/Shuttles/BlackMarket/barnacle.yml",
+            "/Maps/_NF/Shuttles/BlackMarket/bocakillo.yml",
+            "/Maps/_NF/Shuttles/BlackMarket/falcon.yml",
+            "/Maps/_NF/Shuttles/BlackMarket/menace.yml",
+            "/Maps/_NF/Shuttles/BlackMarket/schooner.yml",
+            // Bus
+            "/Maps/_NF/Shuttles/Bus/publicts.yml",
+            // Expedition
+            "/Maps/_NF/Shuttles/Expedition/ambition.yml",
+            "/Maps/_NF/Shuttles/Expedition/anchor.yml",
+            "/Maps/_NF/Shuttles/Expedition/brigand.yml",
+            "/Maps/_NF/Shuttles/Expedition/courserx.yml",
+            "/Maps/_NF/Shuttles/Expedition/dartx.yml",
+            "/Maps/_NF/Shuttles/Expedition/decadedove.yml",
+            "/Maps/_NF/Shuttles/Expedition/dragonfly.yml",
+            "/Maps/_NF/Shuttles/Expedition/gasbender.yml",
+            "/Maps/_NF/Shuttles/Expedition/gourd.yml",
+            "/Maps/_NF/Shuttles/Expedition/pathfinder.yml",
+            "/Maps/_NF/Shuttles/Expedition/rosebudmkii.yml",
+            "/Maps/_NF/Shuttles/Expedition/sprinter.yml",
+            // Nfsd
+            "/Maps/_NF/Shuttles/Nfsd/broadhead.yml",
+            "/Maps/_NF/Shuttles/Nfsd/cleric.yml",
+            "/Maps/_NF/Shuttles/Nfsd/empress.yml",
+            "/Maps/_NF/Shuttles/Nfsd/fighter.yml",
+            "/Maps/_NF/Shuttles/Nfsd/hospitaller.yml",
+            "/Maps/_NF/Shuttles/Nfsd/inquisitor.yml",
+            "/Maps/_NF/Shuttles/Nfsd/interceptor.yml",
+            "/Maps/_NF/Shuttles/Nfsd/marauder.yml",
+            "/Maps/_NF/Shuttles/Nfsd/opportunity.yml",
+            "/Maps/_NF/Shuttles/Nfsd/prowler.yml",
+            "/Maps/_NF/Shuttles/Nfsd/rogue.yml",
+            "/Maps/_NF/Shuttles/Nfsd/templar.yml",
+            "/Maps/_NF/Shuttles/Nfsd/wasp.yml",
+            "/Maps/_NF/Shuttles/Nfsd/whiskey.yml",
+            // Scrap
+            "/Maps/_NF/Shuttles/Scrap/bison.yml",
+            "/Maps/_NF/Shuttles/Scrap/canister.yml",
+            "/Maps/_NF/Shuttles/Scrap/disciple.yml",
+            "/Maps/_NF/Shuttles/Scrap/nugget.yml",
+            "/Maps/_NF/Shuttles/Scrap/orange.yml",
+            "/Maps/_NF/Shuttles/Scrap/point.yml",
+            "/Maps/_NF/Shuttles/Scrap/tide.yml",
+            // Sr
+            "/Maps/_NF/Shuttles/Sr/bottleneck.yml",
+            "/Maps/_NF/Shuttles/Sr/broom.yml",
+            "/Maps/_NF/Shuttles/Sr/mailpod.yml",
+            "/Maps/_NF/Shuttles/Sr/parcel.yml",
+            "/Maps/_NF/Shuttles/Sr/watchdog.yml",
+            // Syndicate
+            "/Maps/_NF/Shuttles/Syndicate/hunter.yml",
+            "/Maps/_NF/Shuttles/Syndicate/infiltrator.yml",
```

Comment:
Dear God no. This needs to be smart and pull the shuttles dynamically, not hard code each one into the test

Reply:
Temp till we pass without fails
After that we can just feed it the folder.